test/test_dss_cli.py

```diff
@@ -172,24 +172,26 @@ def test_get_files_cli(self):
         file_uuid = response['files'][0]['uuid']
 
         process = subprocess.Popen(["hca", "get-files", file_uuid, "--replica", "aws"], stdout=subprocess.PIPE)
-        out, _ = process.communicate()
-        self.assertIsInstance(out, bytes)
+        stdout, stderr = process.communicate()
+        self.assertIsInstance(stdout, bytes)
         with open(file_path, "rb") as bytes_fh:
             file_content = bytes_fh.read()
-            if file_content != out:
-                self.fail("output does not match file content: \"%s\"" % out)
+            if file_content != stdout:
+                self.fail("output does not match file content: \"%s\"" % stdout)
 
     def test_parsing(self):
         """Test that the parser parses arguments correctly."""
 
         cli = hca.dss.cli.CLI()
 
         args = ["put-files", "134", "--bundle-uuid", "asdf", "--creator-uid", "1", "--source-url", "sljdf.com"]
-        out = {'source_url': 'sljdf.com', 'bundle_uuid': 'asdf', 'uuid': '134', 'creator_uid': 1}
-        self.assertEqual(cli.parse_args(args), out)
+        expected = {'source_url': 'sljdf.com', 'bundle_uuid': 'asdf', 'uuid': '134', 'creator_uid': 1}
+        func, api_args = cli.parse_args(args)
+        self.assertEqual(api_args, expected)
```

Comment:
should have a `self.assertEqual(func, "put-files")`

and same for the subsequent cases.

Reply:
func is actually a real function now. e.g. `<bound method AddedCommand.run_from_cli of <class 'hca.dss.autogenerated.put_files.PutFiles'>>`
Can you assert against those?

Reply:
Apparently you can